packages/cf-component-textarea/src/Textarea.js

```diff
@@ -16,22 +16,14 @@ class Textarea extends React.Component {
   }
 
   render() {
-    let className = 'cf-textarea';
+    let {className, ...props} = this.props;
+    
+    className = 'cf-textarea';
```

Comment:
I did it slightly differently but I can change it to match input if that's better.